BestBuyCRUD/EmployeesRepo.cs

```diff
@@ -0,0 +1,168 @@
+                        FirstName = reader["First"].ToString(),
+                        MiddleInitial = reader["Middle"].ToString(),
+                        LastName = reader["Last"].ToString(),
+                        EmailAddress = reader["Email"].ToString(),
+                        PhoneNumber = reader["Number"].ToString(),
+                        Title = reader["Title"].ToString(),
+                        DateOfBirth = Convert.ToDateTime(reader["Bday"])
+                    };
+                    employees.Add(emp);
+                }
+                return employees;
+                
+            }
+         }
+
+        public void CreateEmployees(Employees createEmployees)
+        {
+            MySqlConnection connect= new MySqlConnection(connectionString);
+
+            using (connect)
+            {
+                connect.Open();
+                MySqlCommand cmd = connect.CreateCommand();
+                cmd.CommandText = "INSERT INTO employees (FirstName, MiddleInitial," +
+                    "LastName, EmailAddress, PhoneNumber, Title, DateOfBirth) VALUES" +
+                    "( @First, @Middle, @Last, @Email, @Number,@Title,@Bday);";
+               // cmd.Parameters.AddWithValue("ID", createEmployees.EmployeeId);
+                cmd.Parameters.AddWithValue("First", createEmployees.FirstName);
+                cmd.Parameters.AddWithValue("Middle", createEmployees.MiddleInitial);
+                cmd.Parameters.AddWithValue("Last", createEmployees.LastName);
+                cmd.Parameters.AddWithValue("Email", createEmployees.EmailAddress);
+                cmd.Parameters.AddWithValue("Title", createEmployees.Title);
+                cmd.Parameters.AddWithValue("Number", createEmployees.PhoneNumber);
+                cmd.Parameters.AddWithValue("BDay", createEmployees.DateOfBirth);
+
+                cmd.ExecuteNonQuery();
+            }
+        }
+
+
+        public void UpdateEmployees(int eId,string fn, string ln, string m )
+        {
+            MySqlConnection connect = new MySqlConnection(connectionString);
+            using (connect)
+            {
+                connect.Open();
+                MySqlCommand cmd = connect.CreateCommand();
+                cmd.CommandText = "UPDATE Employees SET FirstName=@First,LastName=@Last, MiddleInitial=@Middle "
+                    + "Where EmployeeId=@ID;";
+
+                cmd.Parameters.AddWithValue("ID", eId);
+                cmd.Parameters.AddWithValue("First", fn);
+                cmd.Parameters.AddWithValue("Last", ln);
+                cmd.Parameters.AddWithValue("Middle", m);
+                // cmd.Parameters.AddWithValue("Email", EmailAddress);
+                // cmd.Parameters.AddWithValue("Phone", DateOfBirth);
+                //cmd.Parameters.AddWithValue("Bday", DateOfBirth);
+
+                cmd.ExecuteNonQuery();
+            } 
+               
+        }
+
+
+        public void DeleteEmployees (int eId)
```

Comment:
"eId" is not a very helpful parameter name. I'm assuming it means Employee Id, but if so then it would be better to go ahead and name it "employeeId".
Also, to be consistent with how the other method signatures look, the space between this method name and the parenthesis should be removed.